grade_school.py

```diff
@@ -0,0 +1,130 @@
+"""Exercise grade-school from exercism.
+https://exercism.org/tracks/python/exercises/grade-school"""
+
+
+class School:
+    """Based on the names of the students and the grade level they are in,
+    the class creates a roster for the school.
+    Grades should be sorted by 1, 2, 3, etc., and students within a grade
+    should be sorted alphabetically by name.
+    Note that all of our students have only have one name and each student can only
+    be added to a grade or roster once."""
+
+    def __init__(self):
+        """Method, which shows us an empty list of students and an additional empty list.
+        Also show us a dictionary (self.grade_list) where the keys are
+        the names of the students and the values correspond to their grades.
+        """
+
+        self.student_list = []
+        self.adding_list = []
+        self.grade_list = {}
+
+    def add_student(self, name, grade):
+        """Method that checks whether a name already exists in the student_list.
+        If not, add it to the student_list, where a boolean True is also added
+        to the adding_list and the name and the grade to the dict grade_list.
+        If not, no name and grade will be added to the student_list and grade_list,
+        but only a boolean False is added to the adding_list.
+
+        Parameters
+        --------
+        name: string
+        grade: int
+        """
+
+        if name not in self.student_list:
+            self.student_list.append(name)
+            self.adding_list.append(True)
+            self.grade_list[name] = grade
+        else:
+            self.adding_list.append(False)
+
+    def roster(self):
+        """Method that provides us with the list of students, sorted by name and grade.
+
+        Returns:
+        list(dict_order.keys()): list[str]
```

Comment:
If the returned variable does not have a name, just write down the type 

```suggestion
        list[str]
```